python/tests/test_parallel_conv.py

```diff
@@ -5,6 +5,9 @@
     from mock import patch, MagicMock
     import numpy as np
     import tensorflow as tf
+    import pytest
+    from eight_mile.utils import get_version
+    pytestmark = pytest.mark.skipif(get_version(tf) >= 2, reason='tf2.0')
```

Comment:
Yeah, these tests were failing for tf2 so they are getting skipped when tf2 is installed, we need to add equivalent tests sometime soon.

The failing might also just be from the way the tests are run because AFAIK the parallel conv code isn't different between tf1 and tf2